cognite/client/_api/simulators/simulator_models.py

```diff
@@ -179,3 +187,136 @@ def retrieve(self, id: int | None = None, external_id: str | None = None) -> Sim
+        )
+
+    def delete(
+        self,
+        id: int | Sequence[int] | None = None,
+        external_ids: str | SequenceNotStr[str] | None = None,
+    ) -> None:
+        """`Delete one or more models <https://api-docs.cognite.com/20230101-beta/tag/Simulator-Models>`_
+
+        Args:
+            id (int | Sequence[int] | None): Id or list of ids
+            external_ids (str | SequenceNotStr[str] | None): No description.
+        Examples:
+
+            Delete models by id or external id::
+
+                >>> from cognite.client import CogniteClient
+                >>> client = CogniteClient()
+                >>> client.simulators.delete_models(id=[1,2,3], external_id="3")
+        """
+        self._delete_multiple(
+            identifiers=IdentifierSequence.load(ids=id, external_ids=external_ids),
+            wrap_ids=True,
+            resource_path="/simulators/models",
+        )
+
+    @overload
+    def create_revisions(self, revision: Sequence[SimulatorModelRevision]) -> SimulatorModelRevisionList: ...
+
+    @overload
+    def create_revisions(
+        self, revision: SimulatorModelRevision | SimulatorModelRevisionWrite
+    ) -> SimulatorModelRevisionList: ...
+
+    def create_revisions(
+        self,
+        revisions: SimulatorModelRevision
+        | SimulatorModelRevisionWrite
+        | Sequence[SimulatorModelRevision]
+        | Sequence[SimulatorModelRevisionWrite],
+    ) -> SimulatorModel | SimulatorModelList:
+        """`Create one or more simulator model revisions. <https://api-docs.cognite.com/20230101-beta/tag/Simulator-Models/operation/create_simulator_model_revision_simulators_models_revisions_post>`_
+
+        You can create an arbitrary number of simulator model revisions, and the SDK will split the request into multiple requests.
+
+        Args:
+            revisions (SimulatorModelRevision | SimulatorModelRevisionWrite | Sequence[SimulatorModelRevision] | Sequence[SimulatorModelRevisionWrite]): No description.
+
+        Returns:
+            SimulatorModel | SimulatorModelList: Created simulator model(s)
+
+        Examples:
+
+            Create new simulator models::
+
+                >>> from cognite.client import CogniteClient
+                >>> from cognite.client.data_classes import SimulatorModelRevision
+                >>> client = CogniteClient()
+                >>> models = [SimulatorModelRevision(external_id="model1"), SimulatorModelRevision(external_id="model2")]
+                >>> res = client.simulators.models.create_revision(models)
+
+        """
+        assert_type(revisions, "simulator_model_revision", [SimulatorModelRevisionCore, Sequence])
+
+        return self._create_multiple(
+            list_cls=SimulatorModelList,
+            resource_cls=SimulatorModel,
+            items=revisions,
+            input_resource_cls=SimulatorModelRevisionWrite,
+            resource_path="/simulators/models/revisions",
+        )
+
```

Comment:
for this operation to work you have to set `limit=1` otherwise this request will fail when multiple models are passed in.